src/components/App.js

```diff
@@ -1,13 +1,13 @@
 import DisplayCalculator from './DisplayCalculator';
 import ButtonPanel from './ButtonPanel';
+import Calculate from '../logic/calculate';
 
 const App = () => (
-  <div className="App">
-    <>
-      <DisplayCalculator />
-      <ButtonPanel />
-    </>
-  </div>
+  <>
+    <DisplayCalculator />
+    <ButtonPanel />
+    <Calculate />
```

Comment:
It appears that you used `Calculate` as a react component. It should be used as a function that other components will rely on to perform calculations. It is therefore better to remove the `<Calculate/>` line.
```suggestion
    <ButtonPanel />
```